rasa/utils/io.py

```diff
@@ -293,6 +293,12 @@ def write_yaml(
     # no wrap lines
     dumper.width = YAML_LINE_MAX_WIDTH
 
+    # use `null` to represent `None`
```

Comment:
`None` isn't mapped to `null` already? :thinking:

Reply:
Somehow no, it wasn't! I've checked the sources, and there's a condition indeed when `null` is replaced by just empty space, but I couldn't quickly figure out what leads to it 😞